src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -1250,4 +1291,27 @@ public void setConnectionProject(IGeppettoProject geppettoProject) throws Geppet
 		this.geppettoProject = geppettoProject;
 	}
 
+	/**
+	 * Sends to the client login user privileges
+	 * @param requestID
+	 */
+	public void checkUserPriviledges(String requestID){
+		String userName = this.geppettoManager.getUser().getLogin();
+		String update = "{\"userName\":" + '"' + userName + '"' +
+						",\"login\":" + this.geppettoManager.isLogin() + ",\"privileges\":[";
+		List<UserPrivileges> privileges = this.geppettoManager.getUser().getUserGroup().getPrivileges();
+		for(UserPrivileges up : privileges){
+			update = update.concat('"'+up.toString()+'"'+",");
+		}
```

Comment:
Similar question here about the string manipulation being used to build up the methods here.  More type safety and error checking can be introduced in the case where objects are being used to make the messages.  Also code with complex string manipulation is often a place where errors creep in.